docky/common/project.py

```diff
@@ -42,16 +44,65 @@ def get_containers(self, service=None):
         return self.project.containers(**kwargs)
 
     def display_service_tooltip(self):
+        infos = self._get_services_info()
         for service in self.project.services:
             labels = service.options.get('labels', {})
             if labels.get('docky.access.help'):
                 # TODO remove after some versions
                 logger.warning(
                     "'docky.access.help' is replaced by 'docky.help'. "
                     "Please update this key in your docker files.")
+            if infos.get(service.name):
+                infos[service.name] += labels.get("docky.url_suffix", "")
+                logger.info(infos[service.name])
             if labels.get('docky.help'):
                 logger.info(labels.get('docky.help'))
 
+    def _get_services_info(self):
+        """ Docker inspect provides all we need to guess ip of docky services
+            Here is a format to retrieve information with
+            a dedicated separator in this format:
+
+            mystring:  as start separator
+            :mystring  as end separator
+        """
+        format_ = "docker inspect --format='name: {{.Name}} :name - " \
+                  "{{range $ip, $_ := .NetworkSettings.Networks}}" \
+                  "ip: {{.IPAddress}}{{$ip}} :ip " \
+                  "{{end}}:{{range $p, $conf := .Config.ExposedPorts}} " \
+                  "port: {{$p}} :port {{end}} :port' $(docker ps -aq)"
+        inspect = os.popen(format_)
```

Comment:
you should use the docker api internally

Reply:
https://github.com/docker/docker-py

Reply:
Nice lib, I didn't know and thanks for this review.
But I not sure if it's a good idea to introduce a new dependency only for a usability feature while standard info from docker is available without it.

It seems you're the first to limit dependencies in projects, isn't it ? ;-)
As opposite if in the future this lib is introduced in this project for more consistent features, then I'll convert this code using the lib without hesitate.

Reply:
It's already a dependency of docker-compose.
I suggest it because it can improve readability.
You could also use plumpumb instead of os. popen
like docker[arg1,arg2, arg3]